sensu-report

```diff
@@ -182,10 +209,12 @@ def print_line(entry, max_line_length):
     print
 
 
-def print_report(data, max_line_length):
+def print_report(data, max_line_length, http_code):
```

Comment:
This is increasing the complexity of this function, I would prefer that there just be a different function for reporting that a client doesn't exist, and then the logic in the main function will be more obvious